quiche/src/recovery/rtt.rs

```diff
@@ -77,17 +80,22 @@ impl RttStats {
     ) {
         self.latest_rtt = latest_rtt;
 
-        if self.first_rtt_sample.is_none() {
+        if !self.has_first_rtt_sample {
             self.min_rtt.reset(now, latest_rtt);
             self.smoothed_rtt = latest_rtt;
+            self.max_rtt = latest_rtt;
             self.rttvar = latest_rtt / 2;
-            self.first_rtt_sample = Some(now);
+            self.has_first_rtt_sample = true;
             return;
         }
 
         // min_rtt ignores acknowledgment delay.
         self.min_rtt.running_min(RTT_WINDOW, now, latest_rtt);
 
+        if latest_rtt > self.max_rtt {
+            self.max_rtt = latest_rtt;
+        }
```

Comment:
```suggestion
self.max_rtt = self.max_rtt.max(latest_rtt);
```

Reply:
Is it truly a readability improvement?

Reply:
I think so since it avoids an if condition and captures the intent of the if condition more precisely (set max_rtt to the max of the two value).

Reply:
done